README.md

```diff
@@ -17,6 +17,7 @@ In order to be able to run some integration tests you should ensure that you hav
 ```awsParams
 /[environment]/Trakx/Shrimpy/ApiClient/ShrimpyApiConfiguration/ApiKey
 /[environment]/Trakx/Shrimpy/ApiClient/ShrimpyApiConfiguration/ApiSecret
+/[environment]/Trakx/Shrimpy/ApiClient/ShrimpyApiConfiguration/FavouriteExchangesAsCsv
 /CiCd/Trakx/Shrimpy/ApiClient/ShrimpyApiConfiguration/ApiKey
```

Comment:
I think these `/CiCd/` lines could be removed since the AWS readme updater already added these lines above.

Reply:
Thanks, gone.